salt/minion.py

```diff
@@ -1389,7 +1389,7 @@ def get_executor(name):
                 ret['out'] = 'nested'
             except Exception:
                 msg = 'The minion function caused an exception'
-                log.warning(msg, exc_info_on_loglevel=logging.DEBUG)
+                log.warning(msg, exc_info_on_loglevel=True)
```

Comment:
Instead of `True` use the log level you wish the traceback to be shown at, if I understand correctly, `logging.WARNING` or instead of `exc_info_on_loglevel` pass `exc_info=True`.